extension/src/providers/providerShared.ts

```diff
@@ -0,0 +1,40 @@
+import { Sexpression } from '../format/sexpression';
+import { ReadonlyDocument } from '../project/readOnlyDocument';
+import * as vscode from	'vscode';
+
+export namespace SearchPatterns {
+	export const LOCALIZES = /^DEFUN$|^DEFUN-Q$|^LAMBDA$/i;
```

Comment:
it should be `CALLABLE`?

Reply:
Thanks for the perspective, I definitively struggled with that one. I settled on LOCALIZES because the one thing they had in common was a formal variable localization header. Maybe this one should turn into DEFINES and the current DEFINES should become CALLABLE? I only say that because LAMBDA doesn't _equally_ fit into the notion of CALLABLE without a SETQ that saves it to a variable. IE, it _could_ be defined in such a way that its recyclable/callable name, but unlikely and we aren't doing enough contextual evaluation to figure that out if it was.